### ISSUE TITLE
Update issue

### ISSUE THREAD

**Issue by enadol** [RamiKrispin/coronavirus#65]
Greetings! Everytime I try use update_dataset(), I get a warning that "coronavirus is in use and will not be updated", or something like that, and the process stops. That happens in RStudio and R Terminal as well, even with a restart of the R Session as indicated. Only after uninstalling the package and reinstalling it, the data is updated succesfully. I'm using the newest Dev version of the package, on Windows. Thanks in advance and best regards,

**Comment by RamiKrispin** [comment]
Hi @enadol , could you please provide the exact error message that you are receiving?

**Comment by enadol** [comment]
Sure. These are all the steps. Many thanks!

![grafik](https://user-images.githubusercontent.com/5300289/103436292-3d863b80-4c1a-11eb-8c88-4a8e9f87ec52.png)

**Comment by RamiKrispin** [comment]
Thanks, could you please send me the output of the `sessionInfo()`?

**Comment by enadol** [comment]
Sure. This is what I get:

sessionInfo()
R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19041)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.1252 
[2] LC_CTYPE=German_Germany.1252   
[3] LC_MONETARY=German_Germany.1252
[4] LC_NUMERIC=C                   
[5] LC_TIME=German_Germany.1252    

attached base packages:
[1] stats     graphics  grDevices
[4] utils     datasets  methods  
[7] base     

other attached packages:
[1] coronavirus_0.3.0.9000

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.5        
 [2] svglite_1.2.3.2   
 [3] tidyr_1.1.2       
 [4] prettyunits_1.1.1 
 [5] class_7.3-17      
 [6] ps_1.5.0          
 [7] assertthat_0.2.1  
 [8] rprojroot_2.0.2   
 [9] digest_0.6.27     
[10] R6_2.5.0          
[11] leafpop_0.0.6     
[12] evaluate_0.14     
[13] e1071_1.7-4       
[14] httr_1.4.2        
[15] ggplot2_3.3.3     
[16] pillar_1.4.7      
[17] gdtools_0.2.2     
[18] rlang_0.4.10      
[19] curl_4.3          
[20] lazyeval_0.2.2    
[21] uuid_0.1-4        
[22] rstudioapi_0.13   
[23] data.table_1.13.6 
[24] callr_3.5.1       
[25] rmarkdown_2.6     
[26] desc_1.2.0        
[27] devtools_2.3.2    
[28] htmlwidgets_1.5.3 
[29] munsell_0.5.0     
[30] tinytex_0.28      
[31] compiler_4.0.3    
[32] xfun_0.19         
[33] pkgconfig_2.0.3   
[34] systemfonts_0.3.2 
[35] base64enc_0.1-3   
[36] pkgbuild_1.2.0    
[37] htmltools_0.5.0   
[38] tidyselect_1.1.0  
[39] tibble_3.0.4      
[40] fansi_0.4.1       
[41] viridisLite_0.3.0 
[42] crayon_1.3.4      
[43] dplyr_1.0.2       
[44] withr_2.3.0       
[45] sf_0.9-6          
[46] grid_4.0.3        
[47] jsonlite_1.7.2    
[48] gtable_0.3.0      
[49] lifecycle_0.2.0   
[50] DBI_1.1.0         
[51] magrittr_2.0.1    
[52] units_0.6-7       
[53] scales_1.1.1      
[54] KernSmooth_2.23-17
[55] cli_2.2.0         
[56] fs_1.5.0          
[57] remotes_2.2.0     
[58] testthat_3.0.1    
[59] ellipsis_0.3.1    
[60] brew_1.0-6        
[61] generics_0.1.0    
[62] vctrs_0.3.6       
[63] tools_4.0.3       
[64] glue_1.4.2        
[65] purrr_0.3.4       
[66] processx_3.4.5    
[67] pkgload_1.1.0     
[68] yaml_2.2.1        
[69] colorspace_2.0-0  
[70] sessioninfo_1.1.1 
[71] classInt_0.4-3    
[72] memoise_1.1.0     
[73] plotly_4.9.2.2    
[74] knitr_1.30        
[75] usethis_2.0.0

**Comment by enadol** [comment]
One more thing: when there's no update available, I get the message as desired, so it appears the warning jumps only when new data is to be updated. Hope this helps. Cheers!

**Comment by RamiKrispin** [comment]
Could you please try to run this and send me the output?

``` r
devtools::install_github("RamiKrispin/coronavirus",
                                   upgrade = "never",
                                   ref = "master")
```

**Comment by enadol** [comment]
Greetings! I first removed the package and then, the following:

devtools::install_github("RamiKrispin/coronavirus", upgrade = "never", ref = "master")
Downloading GitHub repo RamiKrispin/coronavirus@master
√  checking for file 'C:\Users\enado\AppData\Local\Temp\RtmpiKYd84\remotes42b03bf124e6\RamiKrispin-coronavirus-9e83a2a/DESCRIPTION' (380ms)
-  preparing 'coronavirus': (341ms)
√  checking DESCRIPTION meta-information ... 
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'coronavirus_0.3.1.tar.gz'
   
Installing package into ‘C:/Users/enado/Documents/R/win-library/4.0’
(as ‘lib’ is unspecified)
* installing *source* package 'coronavirus' ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
  converting help for package 'coronavirus'
    finding HTML links ... fertig
    coronavirus                             html  
    update_dataset                          html  
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
*** arch - i386
*** arch - x64
** testing if installed package can be loaded from final location
*** arch - i386
*** arch - x64
** testing if installed package keeps a record of temporary installation path
* DONE (coronavirus)

**Comment by enadol** [comment]
However, the data seems to be stalled on January 7th, that is, there's no new data on the repository for January 8 and 9, at least on the csv version. Maybe another issue? Best regards,

**Comment by RamiKrispin** [comment]
there was issue with the git merge of the csv file, fixed and pushed an update. Could you please try again and let me know if you are able to get the data?

**Comment by enadol** [comment]
Yes, the data is now updated. Thanks!

**Comment by RamiKrispin** [comment]
Great! going to push some changes to CRAN in the next few days, in the meanwhile the Github version should provide the same functionality
